framework/generated/generated_vulkan_layer_func_table.h

```diff
@@ -47,696 +47,700 @@
-    { "vkGetAccelerationStructureDeviceAddressKHR",                                                          reinterpret_cast<PFN_vkVoidFunction>(encode::vkGetAccelerationStructureDeviceAddressKHR) },
-    { "vkCmdWriteAccelerationStructuresPropertiesKHR",                                                       reinterpret_cast<PFN_vkVoidFunction>(encode::vkCmdWriteAccelerationStructuresPropertiesKHR) },
-    { "vkGetDeviceAccelerationStructureCompatibilityKHR",                                                    reinterpret_cast<PFN_vkVoidFunction>(encode::vkGetDeviceAccelerationStructureCompatibilityKHR) },
-    { "vkGetAccelerationStructureBuildSizesKHR",                                                             reinterpret_cast<PFN_vkVoidFunction>(encode::vkGetAccelerationStructureBuildSizesKHR) },
-    { "vkCmdTraceRaysKHR",                                                                                   reinterpret_cast<PFN_vkVoidFunction>(encode::vkCmdTraceRaysKHR) },
-    { "vkCreateRayTracingPipelinesKHR",                                                                      reinterpret_cast<PFN_vkVoidFunction>(encode::vkCreateRayTracingPipelinesKHR) },
-    { "vkGetRayTracingCaptureReplayShaderGroupHandlesKHR",                                                   reinterpret_cast<PFN_vkVoidFunction>(encode::vkGetRayTracingCaptureReplayShaderGroupHandlesKHR) },
-    { "vkCmdTraceRaysIndirectKHR",                                                                           reinterpret_cast<PFN_vkVoidFunction>(encode::vkCmdTraceRaysIndirectKHR) },
-    { "vkGetRayTracingShaderGroupStackSizeKHR",                                                              reinterpret_cast<PFN_vkVoidFunction>(encode::vkGetRayTracingShaderGroupStackSizeKHR) },
-    { "vkCmdSetRayTracingPipelineStackSizeKHR",                                                              reinterpret_cast<PFN_vkVoidFunction>(encode::vkCmdSetRayTracingPipelineStackSizeKHR) },
-    { "vkCmdDrawMeshTasksEXT",                                                                               reinterpret_cast<PFN_vkVoidFunction>(encode::vkCmdDrawMeshTasksEXT) },
-    { "vkCmdDrawMeshTasksIndirectEXT",                                                                       reinterpret_cast<PFN_vkVoidFunction>(encode::vkCmdDrawMeshTasksIndirectEXT) },
-    { "vkCmdDrawMeshTasksIndirectCountEXT",                                                                  reinterpret_cast<PFN_vkVoidFunction>(encode::vkCmdDrawMeshTasksIndirectCountEXT) },
-    { "vk_layerGetPhysicalDeviceProcAddr",                                                                   reinterpret_cast<PFN_vkVoidFunction>(vulkan_entry::GetPhysicalDeviceProcAddr) },
-};
+GFXRECON_BEGIN_NAMESPACE(vulkan_entry_layer)
+static const std::unordered_map<std::string, PFN_vkVoidFunction> GetVulkanFuncTableLayer() {
```

Comment:
Should this be inline instead?  Won't every includer have to initialize the map?